alcs-api/src/decision/application-decision/application-decision.dto.ts

```diff
@@ -42,6 +46,14 @@ export class UpdateApplicationDecisionDto {
   @IsBoolean()
   @IsOptional()
   chairReviewOutcome?: boolean | null;
+
+  @IsUUID()
+  @IsOptional()
+  amendsUuid?: string | null;
+
+  @IsUUID()
+  @IsOptional()
+  reconsidersUuid?: string | null;
```

Comment:
just a nitpick, should this also be amendments and reconsiderations for consistency?